src/sentry/models/commitauthor.py

```diff
@@ -20,6 +20,13 @@ class Meta:
 
     __repr__ = sane_repr("organization_id", "email", "name")
 
+    @classmethod
+    def get_or_create(cls, organization_id, email, defaults):
```

Comment:
Could we use a Object Manager here instead, that way if someone adds new code to create commit authors they can still use `objects.get_or_create`?

Reply:
@wmak @wedamija Totally makes sense, I will look into this. But I should point out I did it this way because I saw it being done this way in a number of other models such as `Release`. Were those done incorrectly?

Reply:
IMO its not incorrect to have this as a classmethod, but using an object manager would be better since its less likely for someone to use `get_or_create` incorrectly.